app/src/main/java/com/reach5/identity/sdk/demo/JavaMainActivity.java

```diff
@@ -13,9 +13,9 @@
 import com.reach5.identity.sdk.core.JavaReachFive;
 import com.reach5.identity.sdk.core.Provider;
 import com.reach5.identity.sdk.core.models.AuthToken;
-import com.reach5.identity.sdk.core.models.SdkConfig;
+import com.reach5.identity.sdk.core.SdkConfig;
 import com.reach5.identity.sdk.core.models.OpenIdUser;
-import com.reach5.identity.sdk.core.models.requests.ProfileSignupRequest;
+import com.reach5.identity.sdk.core.models.ProfileSignupRequest;
```

Comment:
I would have preferred that you do a specific PR for your refactoring. It has nothing to do with the refresh token and it implies breaking changes for the customers. You will need to update the documentation and to list all the changes in the changelog file.